core/src/boot.py

```diff
@@ -3,14 +3,43 @@
 import storage.sd_salt
 from trezor import config, io, log, loop, res, ui, utils
 from trezor.pin import pin_to_int, show_pin_timeout
+from trezor.ui.confirm import HoldToConfirm
+from trezor.ui.text import Text
 
 from apps.common.request_pin import PinCancelled, request_pin
 from apps.common.sd_salt import SdProtectCancelled, request_sd_salt
 
 
+async def assert_sdcard() -> None:
+    sd = io.SDCard()
+    if not sd.present():
+        return
+
+    sd.power(True)
+    fs = io.FatFS()
+    try:
+        fs.mount()
+    except OSError:
+        text = Text("SD card", ui.ICON_WRONG, ui.RED)
+        text.normal("Unrecognized filesystem")
+        text.normal("found. Unplug the device")
+        text.normal("and remove the card or")
+        text.normal("you can format the card")
+        text.normal("losing all the data on it.")
+        dialog = HoldToConfirm(text, confirm="Format SD card")
+        await dialog
+        fs.mkfs()
+        fs.setlabel("TREZOR")
```

Comment:
Lowercase? :)

Reply:
This is intentional. See https://superuser.com/questions/137569/setting-a-mixed-case-volume-label-for-a-fat32-drive-on-windows

TL:DR You achieve better compatibility if you use upper-case letters for FAT32 label.